Add Matter firmware update documentation (#33874)
home-assistant/home-assistant.github.io: source/_integrations/matter.markdown

**IP addresses**: Typically more than one IPv6 address is shown: link local, unique local, and global unicast. In some cases a device also supports IPv4. In that case there will also be listed an IPv4 address here.

## Matter device updates

> The Matter standard supports OTA (Over-the-Air) updates optionally. Matter devices which support Matter updates will have an update entity. Furthermore, the CSA DCL (Distributed Compliance Ledger) stores firmware update information. Home Assistant reads firmware update information directly from the DCL to learn about available updates. By default the integration checks every 12 hours for an update. The `homeassistant.update_entity` service call with the update entity as target can be used to force an update check.

Reviewer: ```suggestion
The Matter standard supports OTA (Over-the-Air) updates optionally. Matter devices that support Matter updates will have an update entity. Furthermore, the CSA DCL (Distributed Compliance Ledger) stores firmware update information. Home Assistant reads firmware update information directly from the DCL to learn about available updates. By default. the integration checks every 12 hours for an update. If you want to force an update check, use the `homeassistant.update_entity` [action](/integrations/homeassistant/) with the update entity as the target.
```

Author: Right! Thanks :pray:

**IP addresses**: Typically more than one IPv6 address is shown: link local, unique local, and global unicast. In some cases a device also supports IPv4. In that case there will also be listed an IPv4 address here.

## Matter device updates

> The Matter standard supports OTA (Over-the-Air) updates optionally. Matter devices that support Matter updates will have an update entity. Furthermore, the CSA DCL (Distributed Compliance Ledger) stores firmware update information. Home Assistant reads firmware update information directly from the DCL to learn about available updates. By default, the integration checks every 12 hours for an update. If you want to force an update check, use the `homeassistant.update_entity` [action](/integrations/homeassistant/) with the update entity as the target.

Reviewer 2: ```suggestion
The Matter standard supports OTA (Over-the-Air) updates optionally. Matter devices that support Matter updates will have an [update entity](/integrations/update). Furthermore, the CSA DCL (Distributed Compliance Ledger) stores firmware update information. Home Assistant reads firmware update information directly from the DCL to learn about available updates. By default, the integration checks every 12 hours for an update. If you want to force an update check, use the `homeassistant.update_entity` [action](/integrations/homeassistant/) with the update entity as the target.

```
